CONTRIBUTING-eng.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to PocketFlow
+[Tencent Opensource project](https://opensource.tencent.com/contribution) encourages all developer's participation。You are welcome to [report Issues](https://github.com/Tencent/PocketFlow/issues) or [pull requests](https://github.com/Tencent/PocketFlow/pulls)。 Before contributing, please read the following.
+
+## Debugging and Issue Reporting
+We use Github Issues to track public bugs and feature requests。
```

Comment:
Use English period '.' instead of '。' after "track public bugs and feature requests".